lib/src/snippet/kotlin/com/lemonappdev/konsist/CleanArchitectureSnippets.kt

```diff
@@ -38,9 +40,9 @@ class CleanArchitectureSnippets {
             .classes()
             .withNameEndingWith("UseCase")
             .assert {
-                it.countDeclarations() == 1 && it.containsFunction { function ->
+                it.containsFunction { function ->
                     function.name == "invoke" && function.isPublicOrDefault && function.hasOperatorModifier
-                }
+                } && it.declarationsOf<KoVisibilityModifierProvider>().count() == 1
```

Comment:
` it.declarationsOf<KoVisibilityModifierProvider>().count() == 1` this should check that there is only one declaration with public modifier